### PR TITLE
Modify platform view channel interface

### PR DESCRIPTION
**Pull request by bwikbs**
part of https://github.com/flutter-tizen/plugins/pull/397

### CODE REVIEW

File: shell/platform/tizen/channels/platform_view_channel.cc [hunk 1]
```diff
@@ -229,7 +229,7 @@ void PlatformViewChannel::OnResize(
   }
   view->Resize(*width, *height);
 
-  result->Success();
+  result->Success(*arguments);
```
Comment (bbrto21) [inline shell/platform/tizen/channels/platform_view_channel.cc:232]: Don't you need some logic as below?

```java
            final PlatformViewBufferSize sz = handler.resize(resizeRequest);
            if (sz == null) {
              result.error("error", "Failed to resize the platform view", null);
            } else {
              final Map<String, Object> response = new HashMap<>();
              response.put("width", (double) sz.width);
              response.put("height", (double) sz.height);
              result.success(response);
            }
```

Comment (bbrto21) [inline shell/platform/tizen/channels/platform_view_channel.cc:232]: Don't you need some logic as below?

```java
            final PlatformViewBufferSize sz = handler.resize(resizeRequest);
            if (sz == null) {
              result.error("error", "Failed to resize the platform view", null);
            } else {
              final Map<String, Object> response = new HashMap<>();
              response.put("width", (double) sz.width);
              response.put("height", (double) sz.height);
              result.success(response);
            }
```

Comment (bwikbs) [reply shell/platform/tizen/channels/platform_view_channel.cc:232]: You are right, but in the current state, I guess we don't need it. 
It is true that readability is poor.

Comment (bwikbs) [reply shell/platform/tizen/channels/platform_view_channel.cc:232]: Is this the same as above? :smile

**Comment by bbrto21** [comment]
I don't know why my review keeps being posted many times. :cry:

**Review by bbrto21 — APPROVED** [review]

**Review by swift-kim — APPROVED** [review]

**Pull request merged**